docs/test-page/index.md

```diff
@@ -0,0 +1,362 @@
+      <div class="col-12 col-lg-3">
+        {%- include 'markup/input-text.njk' -%}
+      </div>
+      <div class="col-12 col-lg-3">
+        {%- include 'markup/select.njk' -%}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        {%- include 'markup/sample-table.njk' -%}
+      </div>
+    </div>
+    <div class="row mb-n6">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    {%- include 'markup/pagination.njk' -%}
+  </div>
+</div>
+
+<hr class="major">
+
+<!-- typography and toc -->
+
+<div class="content-container">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12 col-lg-4 order-lg-2">
+        {%- include 'markup/sample-toc.njk' -%}
+      </div>
+      <div class="col-12 col-lg-8 order-lg-1">
+        {%- include 'markup/sample-typography.njk' -%}
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <hr class="minor">
+      </div>
+    </div>
+    {%- include 'markup/button-page-actions.njk' -%}
+  </div>
+</div>
+
+<!-- modals and toasts buttons -->
+
+<div class="content-container">
+  <div class="container-fluid" role="group" aria-labelledby="form-section-01-description">
+    <div class="row">
+      <div class="col-12">
+        <h2 class="legend">Modals and Toasts</h2>
+      </div>
+    </div>
+  </div>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-12">
+        {%- include 'markup/modal.njk' -%}
+        <button type="button" class="btn btn-info">Launch Info Toast</button>
+        <button type="button" class="btn btn-danger">Launch Danger Toast</button>
+        <button type="button" class="btn btn-success">Launch Success Toast</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<hr class="major">
+
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-12 col-lg-4">
+      <div class="card">
+        <div class="card-body">
+          <p class="card-title">Card Title</p>
+          <p class="mb-0">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin varius blandit nisl. Nulla non magna nec risus vehicula aliquet. Ut feugiat sem ut euismod tempus. Quisque tincidunt finibus felis eu tempus. Sed mauris risus, elementum eleifend leo eu, accumsan efficitur lacus. Maecenas quis massa a nisi varius pulvinar. Cras rhoncus eu mauris vel vehicula. Vivamus blandit, tellus sit amet consectetur luctus, nisl neque viverra urna, eu finibus urna magna ac dui. Nulla nulla est, posuere sed sem at, eleifend pulvinar nunc. In tempor urna id massa eleifend hendrerit.</p>
+        </div>
+      </div>
+    </div>
+    <div class="col-12 col-lg-4">
+      <div class="data-tile">
+        <div class="data-tile-body">
+          <p class="fw-bold text-black mb-0">Label</p>
+          <hr class="minor my-2">
+          <p class="mb-0">Value</p>
+        </div>
+      </div>
+      {%- include 'markup/progress-small.njk' -%}
+    </div>
```

Comment:
There is no bottom margin spacing at mobile sizes between the progress-small and the share-social partials.

Reply:
good catch